### PR TITLE
Support the bootstrap modal

### PR DESCRIPTION
**Pull request by iflamed**
Because the bootstrap also use a backdrop, when use the selectivity in
the bootstrap modal, it can not run normally.

### CODE REVIEW

File: dist/selectivity-full.js [hunk 1]
```diff
@@ -360,7 +360,8 @@ $.extend(SelectivityDropdown.prototype, {
         } else {
             $backdrop = $('<div>').addClass('selectivity-backdrop');
 
-            $('body').append($backdrop);
+            this.$el.prepend($backdrop);
+            this.$el.children().not('.selectivity-backdrop').css('z-index',9999);
```
Comment (iflamed) [inline dist/selectivity-full.js]: May this can be fixed in the css. But I don't know which css class need be updated. So just use the javascript.

**Comment by msx2** [comment]
@iflamed I've tested the current behavior of the selectivity inputs in bootstrap modal and it seems to work as I would expect it to, i.e. `.selectivity-backdrop` is the element on top of the `.modal-backdrop`, so basically in order to close modal you need to perform two clicks - one to close the input, the other one to close/access the modal.

If you want to prioritize `.modal`, i.e. be able to close the modal while selectivity input is still open, or select any other element inside of it, you should rather do this in *your own* styles by increasing `z-index` on `.modal`. By default `.selectivity-backdrop` comes with `z-index: 9998`, so the value of `9999` will suffice for `.modal` to be accessible. Or you can use a lower `z-index` for `.selectivity-backdrop` - bootstrap's default for `.modal` is `1050` and `1040` for `.modal-backdrop`.

Also, `z-index` property is inherited. In your solution, if `$this.$el` has `z-index: 10`, the backdrop inside will not get anything more than that :wink:

**Comment by iflamed** [comment]
@msx2 
hi, I wrote a fiddle to show you about this bug. https://jsfiddle.net/bnu_oxkiller/0tx3fd13/embedded/result/

How it going:
1. Launch the modal
2. Then click the input to choose a name.
3. Select a choice, then you will found it can not be choose.

**Comment by msx2** [comment]
@iflamed Okay, you are right. Sorry. There's no physical way of selecting the option by clicking, and only using keyboard. That's due to inherited `z-index: 1050` from `.modal`.

A quick workaround for this would be to add this to your local styles: `.selectivity-backdrop { z-index: 1045; }`. A more generic solution can be found in #50.

**Comment by iflamed** [comment]
@msx2  I will close the pr.
Solution by the pr #50  is much more clean and nice. Please merge #50 instead.

**Pull request closed without merging**